deploy/deduplication-db-migrations/chart/templates/migrations-job.yaml

```diff
@@ -20,7 +20,7 @@ spec:
       restartPolicy: Never
       containers:
       - name: enrichment-db-migrations
-        image: "{{ default "ghcr.io/ocurity/dracon" .Values.image.registry }}/draconctl:{{ default .Chart.AppVersion .Values.image.tag }}"
+        image: "{{ default "ghcr.io/ocurity/dracon" .Values.global.image.registry }}/ocurity/dracon/draconctl:{{ default .Chart.AppVersion .Values.image.tag }}"
```

Comment:
this now doesn't work if the default is used, then it'd be `ghcr.io/ocurity/dracon/ocurity/dracon/draconctl...`

Reply:
u can just move that over from `deploy/dracon/values` and remove it there, it's only needed for the migrations